content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/autocomponents.md

```diff
@@ -98,3 +91,19 @@ The following is an example of an auto-component which synchronizes a component
     </RemoteProcedure>
 </Component>
 ```
+
+## Building auto-components
+
+Auto-components are processed when you compile and build your project.
+Make sure to add your auto-component files inside your project's CMake file so they can be built.
+
+This example of `<your-project>_files.cmake` lists the auto-component files.
+```cmake
+set(FILES
+    ...
+    Source/AutoGen/NetworkTestPlayerComponent.AutoComponent.xml
+    Source/AutoGen/MySimpleNetPlayerComponent.AutoComponent.xml    
+)    
+```
+
+Any time that you update a CMake file, you must reconfigure and recompile O3DE Editor, Game Launcher, and Server Launcher. For more information, refer to [Configure and Build](/docs/user-guide/build/configure-and-build).
```

Comment:
@AMZN-Gene @o3de/sig-network 
If we make changes to the `*.AutoComponent.xml` file, does that hot reload, or do we still need to rebuild the project? I'd like to clarify that here.

Reply:
Good question, and yes! Changing the XML (adding a network property, adding an RPC, etc) will require a re-compile because that XML is used to generate (AzCodeGen) a C++ file.